pkgs/development/python-modules/refery/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+
+, poetry-core
+, pyyaml
+, colorama
+, junit-xml
+}:
+
+buildPythonPackage rec {
+  pname = "refery";
+  version = "2.1.0";
+  format = "pyproject";
+
+  src = fetchPypi {
```

Comment:
Can you instead build from source? I think it should be a trivial modification

Reply:
What do you mean with building from source, the only difference between fetching the src from pypi or github is that github includes tests, but it doesn't matter since the package doesn't have any. And since python interpreted anyways there isn't really a building from source.